mesonbuild/compilers.py

```diff
@@ -2042,6 +2042,15 @@ def get_option_link_args(self, options):
             return options['c_winlibs'].value
         return []
 
+    @staticmethod
+    def get_gcc_version(executable):
```

Comment:
Maybe just call this `get_version` and put it on the `GnuCompiler` class since it's for all GNU compilers.